abr_analyze/__init__.py

```diff
@@ -6,3 +8,10 @@
 from . import gui
 
 from .data_handler import DataHandler
+
+dir_name = os.path.dirname(__file__)
```

Comment:
is this writing a file with these lines for the user to fill in?

Reply:
This creates the text files to create your own custom paths for the db, images etc. I made them generated instead of having them live in the repo to avoid running into conflicts when you change the file to your own path. If the files don't exist it will create them. In paths.py it checks those files first to see if any of the paths have been set. If not it will use the default paths.